backend/src/main/java/gov/cdc/usds/simplereport/service/sms/TextMessageStatusService.java

```diff
@@ -29,7 +29,7 @@
 @Transactional
 public class TextMessageStatusService {
 
-  private PhoneNumberRepository _phoneRepo;
+  private PhoneNumberRepository phoneRepo;
```

Comment:
Also fixed a couple of style issues in different classes - _underscore naming is no longer convention.